src/main/scala/mesosphere/marathon/core/launchqueue/impl/TaskLauncherActor.scala

```diff
@@ -201,7 +201,7 @@ private class TaskLauncherActor(
       logger.debug(s"After delay update $status")
 
     case msg @ RateLimiter.DelayUpdate(ref, delayUntil) if ref != runSpec.configRef =>
-      logger.warn(s"BUG! Received delay update for other run spec ${ref} and delay $delayUntil. Current run spec is ${runSpec.configRef}")
+      logger.warn(s"Ignoring delay update for old run spec $ref and delay $delayUntil. Current run spec is ${runSpec.configRef}")
```

Comment:
I thought it *is* a bug 😉. In any case this should go with #6664.

Reply:
hmm no, when we are in the middle of replace, I think it's expected behavior :)